packages/@sanity/base/src/_exports/_internal.ts

```diff
@@ -11,7 +11,8 @@ export {observePathsDocumentPair as unstable_observePathsDocumentPair} from '../
 // eslint-disable-next-line camelcase
 export {observeDocumentPairAvailability as unstable_observeDocumentPairAvailability} from '../preview/availability'
 
-export {DocumentAvailability, AvailabilityReason} from '../preview/types'
+export type {DocumentAvailability} from '../preview/types'
+export {AvailabilityReason} from '../preview/types'
```

Comment:
```suggestion
export {type AvailabilityReason, DocumentAvailability} from '../preview/types'
```

Reply:
This one doesn't seem like a valid way to do the export.

Reply:
it is valid typescript, but maybe other parsers haven't caught up yet

Reply:
Must be `export type {AvailabilityReason, DocumentAvailability} from '../preview/types'`